docs/samples/Microsoft.ML.Samples/Dynamic/OneHotEncodingTransform.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Collections.Generic;
+using Microsoft.ML.Data;
+using static Microsoft.ML.Transforms.OneHotEncodingTransformer;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class OneHotEncodingTransform
```

Comment:
>OneHotEncodingTransform [](start = 24, length = 23)

Please rename this to OneHotEncoding (identical to the API extension method that it correspond to)